### ISSUE TITLE
bitnami wildfly helm chart volume mount issue 

### ISSUE THREAD

**Issue by chirathb** [bitnami/charts#6701]
helm chart- bitnami/wildfly helm chart

we used bitnami/wildfly helm chart for deploye wildfly service. after enable pv, pod is not creating correctly (pod was Crashloopbackoff ) we used bitnami/wildfly location as mount the path. we used nfs pv. 

@Mauraza

**Comment by carrodher** [comment]
Hi, in order to reproduce the issue on our side, can you share with us the version of the chart you are using as well as the parameters you used during the installation? Did you modify the _values.yaml_ or pass them via `--set`? In any case, it would be useful for us if you can share those customizations.

**Comment by chirathb** [comment]
after volumePermissions true, we can fix that issue. But we can't access Administration Console. 

Issue
Unable to redirect.
An automatic redirect to the Administration Console is not currently available. This is most likely due to the administration console being exposed over a network interface different from the one to which you are connected to.

To access the Administration console you should contact the administrator responsible for this WildFly installation and ask them to provide you with the correct address.

**Comment by carrodher** [comment]
We were able to reproduce the issue on our side, we created an internal task to investigate and fix the issue. Unfortunately and due to the capacity of the team and other priorities I can't provide an ETA for this task.

I'm adding the on-hold label to this issue to prevent our automation from closing the issue due to lack of activity. We will update it with more news once we started working on the task. Thanks.
